.github/workflows/linux.yml

```diff
@@ -152,11 +157,11 @@ jobs:
       - name: Download release artifact
         uses: actions/download-artifact@v4
         with:
-          name: linux-amd64-double-zipped
+          name: linux-amd64-double-zipped-${{ needs.build-linux.outputs.version }}
       - name: Extract release tarball
         run: |
           mkdir -p ~/lib
-          tar -C ~/lib -xf tinygo.linux-amd64.tar.gz
+          tar -C ~/lib -xf tinygo*.linux-amd64.tar.gz
```

Comment:
Any reason we're globbing here instead of using the `...outputs.version` variable?

Reply:
No, it was just easier at the time. Fixed.